flatbuffers:x86-windows cmake Error at ports/flatbuffers/portfile.cmake:46

Package: flatbuffers:x86-windows
 Vcpkg version: 0.0.76-a92ecac8a79c9b03ba3aa86b1478f75c4d2f8847

Builds ok, but CMake Error at ports/flatbuffers/portfile.cmake:46 (file):
  file RENAME failed to rename
  C:/vcpkg/packages/flatbuffers_x86-windows/bin/flatc.exe
to
  C:/vcpkg/packages/flatbuffers_x86-windows/tools/flatc.exe
because: No such file or directory

I assure the exist of  "C:/vcpkg/packages/flatbuffers_x86-windows/bin/flatc.exe".

--- a/ports/flatbuffers/portfile.cmake
+++ b/ports/flatbuffers/portfile.cmake
@@ -37,12 +37,14 @@ vcpkg_configure_cmake(
     # OPTIONS_DEBUG -DDEBUGGABLE=1
 )
 
+vcpkg_build_cmake()
 vcpkg_install_cmake()
 
 if(EXISTS ${CURRENT_PACKAGES_DIR}/debug/bin)
     file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/bin)
 endif()
 if(EXISTS ${CURRENT_PACKAGES_DIR}/bin/flatc.exe)
+    make_directory(${CURRENT_PACKAGES_DIR}/tools)
     file(RENAME ${CURRENT_PACKAGES_DIR}/bin/flatc.exe ${CURRENT_PACKAGES_DIR}/tools/flatc.exe)
     file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/bin)
 endif()

